specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/alter.configs/alter.config/client.rpt

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect "zilla://streams/app0"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .request()
+                                 .alterConfigs()
+                                    .resource()
+                                       .type("TOPIC")
+                                       .name("events")
+                                       .config("cleanup.policy", "delete")
+                                       .validateOnly("false")
+                                       .build()
+                                    .resource()
+                                       .type("TOPIC")
+                                       .name("snapshots")
+                                       .config("cleanup.policy", "compact")
+                                       .validateOnly("false")
+                                       .build()
+                                   .build()
+                                .build()
+                               .build()}
+
+connected
+
+read zilla:begin.ext ${kafka:matchBeginEx()
+                               .typeId(zilla:id("kafka"))
+                               .response()
+                                 .alterConfigs()
+                                    .resource()
+                                       .code(0)
```

Comment:
Perhaps `error` or `errorCode` instead of `code` as `code` seems to lose the intent of it being related to an `error`?